docs/sources/mimir/manage/tools/mimir-continuous-test.md

```diff
@@ -11,32 +11,38 @@ weight: 30
 
 # Grafana mimir-continuous-test
 
-As a developer, you can use the standalone mimir-continuous-test tool to run smoke tests on live Grafana Mimir clusters.
+As a developer, you can use the mimir-continuous-test tool to run smoke tests on live Grafana Mimir clusters.
 This tool identifies a class of bugs that could be difficult to spot during development.
 Two operating modes are supported:
 
 - As a continuously running deployment in your environment, mimir-continuous-test can be used to detect issues on a live Grafana Mimir cluster over time.
 - As an ad-hoc smoke test tool, mimir-continuous-test can be used to validate basic functionality after configuration changes are made to a Grafana Mimir cluster.
 
-## Download mimir-continuous-test
+## Download and Run mimir-continuous-test
```

Comment:
The style guide for Grafana docs prefers sentence case: https://grafana.com/docs/writers-toolkit/write/markdown-guide/#:~:text=Write%20in%20sentence%20case%20throughout%20all%20technical%20documentation%2C%20be%20it%20long%2Dform%20text%20or%20microcopy%20within%20a%20UI%3A

```suggestion
## Download and run mimir-continuous-test
```